### PR TITLE
MORE AI SPRITES OK?

### PR DESCRIPTION
**Pull request by NanoCats**
# About The Pull Request

- Adds a handful of Cool™ and Good™ sprites lovingly hand-stolen from /vg/station, 5 or 6 of them. 
- Mostly I just wanted the Anime Girl one because I like being cute but I found some other cool ones on my search so why the fuck not
- I hate every part of interacting with github
- I actually tested it and it works now although my sanity is heavily eroded
## Why It's Good For The Game

Variety is the something of something, some of these are really fuckin cool.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image
*I would have made this an animated gif if I cared and knew how 
† "Boy" skin not shown because it's pretty much identical to "Girl" with shorter hair and not smiling.

</details>

## Changelog

:cl:

imageadd: Added "Boxfort" "Terminal" "Girl" "Boy" "Hotdog" and "Yesman" skins for AI Enrichment purposes

/:cl:

### CODE REVIEW

File: code/_globalvars/lists/flavor_misc.dm [hunk 1]
```diff
@@ -111,7 +111,17 @@ GLOBAL_LIST_INIT(ai_core_display_screens, sortList(list(
 	"Too Deep",
 	"Triumvirate",
 	"Triumvirate-M",
-	"Weird")))
+	"Weird",
```
Comment (Zonespace27) [inline code/_globalvars/lists/flavor_misc.dm:114]:
```suggestion
    //MONKESTATION ADDITION START
	"Weird",
```
modular comments

File: code/_globalvars/lists/flavor_misc.dm [hunk 2]
```diff
@@ -111,7 +111,17 @@ GLOBAL_LIST_INIT(ai_core_display_screens, sortList(list(
...
+	"Girl",
+	"Girl-malf",
+	"Boy",
+	"Boy-malf",
+	"Yes-man",
+	"Boxfort",
+	"Boxfort-crash",
+	"Hotdog",
+	"Terminal",
+	)))
```
Comment (Zonespace27) [inline code/_globalvars/lists/flavor_misc.dm:124]:
```suggestion
	)))
	//MONKESTATION ADDITION END
```

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
not a fan of the locker perspective cardboard box sprite, but eh

**Comment by NanoCats** [comment]
Ok it turns out the issue was client side it should be AoK now.

**Comment by NanoCats** [comment]
> ai-boxfort-crash needs to be modified to ai-boxfort_dead inside ai.dmi

it is done

**Review by nednaZ — APPROVED** [review]
Looks good to me.
The boxfort one is a bit weird, but it works.

**Comment by NanoCats** [comment]
> Looks good to me. The boxfort one is a bit weird, but it works.

Listen, really I just wanted this adorable thing but I decided to just hoover up the other ones that looked neat so it wasn't as much of a "snowflake" pr and also you gotta admit it's funny. 

Sprite-0001-export

**Pull request merged**